src/View/Helper/PaginatorHelper.php

```diff
@@ -1121,4 +1121,31 @@ public function implementedEvents()
     {
         return [];
     }
+
+    /**
+     * Dropdown array for Select limit
+     *
+     * @param array $limits This is array of options.
+     * @param array $default Default limit for option selecting. Default is Paginator limit.
+     *
+     * @return string html output.
+     */
+    public function limitSelect($limits = [], $default = 0)
+    {
+        $out = $this->Form->create('', ['type' => 'get']);
+
+        if (empty($limits)) {
+            $limits = [10 => $this->Number->format(10), 25 => $this->Number->format(25), 50 => $this->Number->format(50), 100 => $this->Number->format(100)];
```

Comment:
To ensure internationalization. I am using bn_BD as default_locale, so I need the number to show as Bangla font.